doc/api/deprecations.md

```diff
@@ -3171,10 +3171,13 @@ Use [`diagnostics_channel.subscribe(name, onMessage)`][] or
 [`diagnostics_channel.unsubscribe(name, onMessage)`][] which does the same
 thing instead.
 
-### DEP0164: `process.exit([code])` coercion to integer
+### DEP0164: `process.exit([code])`, `process.exitCode` coercion to integer
```

Comment:
I understand this was there prior to this PR, but actually, the deprecation only applies if an argument was provided, so it feels a bit weird to me to mark it as optional. Anyway, feel free to disagree and ignore that suggestion.
```suggestion
### DEP0164: `process.exit(code)`, `process.exitCode` coercion to integer
```

Reply:
Applied the review. Thanks for catching!